### PR TITLE
Create Duplicates For Comparison and Translation

### PR DESCRIPTION
**Pull request by Klarkin2**

### CODE REVIEW

File: src/utils/dynamo.py [hunk 1]
```diff
@@ -22,11 +21,28 @@ def write_to_dynamo(
...
+    existingItem = get_existing_object(country)
+
+    # If we're testing we're find to GET and object out of dynamo
+    # But we don't want to overwrite the 'old' version of the object that is used for comparison
+    # (which is used to tell us if we need to translate it or not)
+    if isTesting:
+        existingItem["country"]["S"] = existingItem["country"]["S"] + testSuffix
+
+    # Mark the existing item as 'old', then scrape to get the most 'up to date' information.
+    update_existing_item_as_old(existingItem)
```
Comment (xlanor) [inline src/utils/dynamo.py:44]: If the code fails at a later part in this function, how do we perform a rollback?

Comment (Jachdich) [reply src/utils/dynamo.py:44]: As far as I can see, there's not much to fail (unless inserting the new values into the db fails, which would mean rolling back wouldn't be possible anyway). Also I'm pretty sure it just makes a copy of the current db entry, so all the data should still be present after the copy.

Comment (Klarkin2) [reply src/utils/dynamo.py:44]: Performing rollback would be as easy as removing '-old' from the object in dynamo. The real problem is, how do we know it failed? We'll have to figure that out sometime soon probably

Comment (xlanor) [reply src/utils/dynamo.py:44]: I think down the line we can explore using something like [PyDynamoDB](https://github.com/pynamodb/PynamoDB) for an ORM in the future, I'm ok to merge first

**Pull request merged**